### ISSUE TITLE
Closing/minimising to tray w/o tray icon visible makes it impossible to get qTox back

### ISSUE THREAD

**Issue by bricky149** [qTox/qTox#746]
...unless we murder the process.

**Comment by dimgel** [comment]
Just issued `emerge @live-rebuild` to get qtox up to date and got same issue: systray icon disappeared. I configured qtox to start minimized to systray, so now I cannot access it at all. Still, it's doing something on its own:

$ kdesudo -u me-qtox qtox
Settings: Loading from  "/home/me-qtox/.config/tox/settings.ini"
Using embeded bootstrap nodes list
built on:  00:29:14 Nov 15 2014 ( 1416000547 )
commit:  7c6f233d537613f56ef9d9b969242c83ded02cf3 

IPC: Our global ID is  1420937645306963688
IPC: Attaching to the global shared memory
Core: loading Tox from "/home/me-qtox/.config/tox/dimgel@qtox.tox"
libGL error: failed to load driver: i965
libGL error: Try again with LIBGL_DEBUG=verbose for more details.
Core starting with IPv6 enabled
Core::loadConfiguration: reading from  "/home/me-qtox/.config/tox/dimgel@qtox.tox"
Core: Error loading self avatar
IPC: Previous owner timed out, taking ownership
Core: Bootstraping to the DHT ...
"Core: Bootstraping from Nikolai Toryzin, addr 192.254.75.98, port 33445"
"Core: Bootstraping from sonOfRa, addr 144.76.60.215, port 33445"
Core: Connected to DHT
Settings: Saving in  "/home/me-qtox/.config/tox/settings.ini"
Core: writing tox_save to  "/home/me-qtox/.config/tox/dimgel@qtox.tox"
Widget: something set the status to online

**Comment by tux3** [comment]
qTox starts without a tray icon from now on because it doesn't work well on Ubuntu systems, but you can still enable it in the options.

Obviously we still need to fix this bug.

You can enter "rm /home/me-qtox/.config/tox/settings.ini" in a terminal if you want to reset the settings, it'll make qTox appear again.

**Comment by dubslow** [comment]
This is one issue I'll fix.

**Comment by strk** [comment]
I've always had this problem on Ubuntu 14.04, with gnome-panel. But earlier today unexpectedly I saw the tox icon in the tray, as of 85aca2ca4004854bfc0c2dc762b37d5dfac80dbe

Killing the gnome-panel made it restart and this time the tox icon wasn't there anymore.
Restarting qTox did not install again either.

Now I pulled and built 546b90c4c9d22a79db2a804e007112d88ea1756a but again it does not install (will try enabling via menu)
